### DISCUSSION TITLE
Html tags in Notifications view

### DISCUSSION THREAD

**Discussion by Globulopolis** [community/community#110166] [Mobile] [answered]
### Select Topic Area

Bug

### Body

Hi!
Some html tags are stripped(or converted) incorrectly in Notifications view.

![20240226_113224](https://github.com/community/community/assets/740757/14ff4139-6e7f-483f-a2d8-96beee3071cd)

**Comment by stevepopovich** [comment, marked as answer]
Thanks for writing in @Globulopolis . I passed this onto the team for consideration.
